src/encoding/text.rs

```diff
@@ -336,6 +342,15 @@ impl<'a, 'b> MetricEncoder<'a, 'b> {
 
         Ok(())
     }
+
+    fn write_timestamp(&mut self, timestamp: SystemTime) -> Result<(), std::fmt::Error> {
+        if let Ok(time) = timestamp.duration_since(UNIX_EPOCH) {
+            self.writer.write_char(' ')?;
+            return self.writer.write_str(&time.as_millis().to_string());
```

Comment:
This library follows the OpenMetrics specification. This should be compatible with recent Prometheus versions (remember to set the right content type).

Please use seconds instead of milliseconds here.

https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#timestamps-1